lua/lualine/components/diff/git_diff.lua

```diff
@@ -123,13 +125,17 @@ end
 
 ---update git_diff variable
 function M.update_git_diff()
-  if M.diff_args then
+  -- It's important here that we check is_running, since
+  -- calling stop() on a git operation can have index.lock remain
+  -- and block other git operations
+  if M.diff_args and not is_running then
```

Comment:
won't this lead to bugs like diff component not updating properly?

Reply:
Possibly, yeah, so maybe it should use a flag to auto run it again in on_exit, when it is still running in update_git_diff